### PR TITLE
Web console: Remove first / last suggestions

### PR DESCRIPTION
**Pull request by vogievetsky**
Comment out some optimistic suggestion until such a time when they will be possible at ingestion time.

Related to https://github.com/apache/druid/issues/10702

image

### CODE REVIEW

File: web-console/src/druid-models/metric-spec.tsx [hunk 1]
```diff
@@ -67,14 +67,17 @@ export const METRIC_SPEC_FIELDS: Field<MetricSpec>[] = [
...
+      // Ref: https://druid.apache.org/docs/latest/querying/aggregations.html#first--last-aggregator
+      // Should the first / last aggregators become usable at ingestion time, comment in the lines below:
+      // {
+      //   group: 'first',
+      //   suggestions: ['longFirst', 'doubleFirst', 'floatFirst'],
+      // },
+      // {
+      //   group: 'last',
+      //   suggestions: ['longLast', 'doubleLast', 'floatLast'],
+      // },
```
Comment (jihoonson) [inline web-console/src/druid-models/metric-spec.tsx:80]: We don't usually allow commented out code in java code. Do you think it's better to be consistent in the web-console as well?

**Review by jihoonson — APPROVED** [review]
LGTM. I left a nit comment on the code style.

File: web-console/src/druid-models/metric-spec.tsx (hunk 1 above)
Comment (vogievetsky) [reply web-console/src/druid-models/metric-spec.tsx]: fixed.

**Comment by suneet-s** [comment]
I don't think this fixes #10702 IMO it should still be an open issue, this fix makes it less likely that someone will trip on this from the UI

**Comment by FrankChen021** [comment]
@suneet-s I'm working on #10702. I have finished main coding and unit test cases, and is going to add new IT test cases. But it will take me some time to finish it since I don't get a large chunk of time these days. Once the fix is completed, I will bring back the UI removed by this PR.

**Comment by suneet-s** [comment]
> @suneet-s I'm working on #10702. I have finished main coding and unit test cases, and is going to add new IT test cases. But it will take me some time to finish it since I don't get a large chunk of time these days. Once the fix is completed, I will bring back the UI removed by this PR.

@FrankChen021 that's great news! Looking forward to the PR!

**Pull request merged**